resources/zib/zib-Alert.xml

```diff
@@ -0,0 +1,203 @@
+      <alias value="Conditie" />
+      <type>
+        <code value="Reference" />
+        <targetProfile value="http://hl7.org/fhir/StructureDefinition/Resource" />
+        <targetProfile value="http://nictiz.nl/fhir/StructureDefinition/zib-Problem" />
+      </type>
+      <mapping>
+        <identity value="zib-alert-v4.1-2020EN" />
+        <map value="NL-CM:8.3.3" />
+        <comment value="Condition" />
+      </mapping>
+    </element>
+    <element id="Flag.extension:comment">
+      <path value="Flag.extension" />
+      <sliceName value="comment" />
+      <max value="1" />
+      <type>
+        <code value="Extension" />
+        <profile value="http://nictiz.nl/fhir/StructureDefinition/ext-Comment" />
+      </type>
+    </element>
+    <element id="Flag.extension:comment.value[x]">
+      <path value="Flag.extension.value[x]" />
+      <short value="Comment" />
+      <definition value="Explanatory comments to the alert that can not be expressed in any of the other elements." />
+      <alias value="Toelichting" />
+      <mapping>
+        <identity value="zib-alert-v4.1-2020EN" />
+        <map value="NL-CM:8.3.7" />
+        <comment value="Comment" />
+      </mapping>
+    </element>
+    <element id="Flag.status">
+      <path value="Flag.status" />
+      <comment value="Sending systems that don't record an explicit status can use the following guidance to infer a value from the zib:&#xD;&#xA;&#xD;&#xA;* EndDateTime present and in the past: _inactive_&#xD;&#xA;* EndDateTime absent: _active_&#xD;&#xA;* EndDateTime present and in the future - goes against both FHIR and zib definitions, but in a case where this might occur: _active_" />
+      <mapping>
+        <identity value="zib-alert-v4.1-2020EN" />
+        <map value="NL-CM:8.3.5" />
+        <comment value="StartDateTime (implicit, main mapping is on Flag.period.start)" />
+      </mapping>
+      <mapping>
+        <identity value="zib-alert-v4.1-2020EN" />
+        <map value="NL-CM:8.3.8" />
+        <comment value="EndDateTime (implicit, main mapping is on Flag.period.end)" />
+      </mapping>
+    </element>
+    <element id="Flag.category">
+      <path value="Flag.category" />
+      <slicing>
+        <discriminator>
+          <type value="value" />
+          <path value="$this" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+    </element>
+    <element id="Flag.category:alertName">
```

Comment:
I think you should rename this to alertType, because alertName is mapped on Flag.code. @ArdonToonstra

Reply:
Good find!